src/actions.rs

```diff
@@ -57,82 +57,85 @@ impl<'a> Action for Step<'a> {
+                                                .labels
+                                                .iter()
+                                                .map(|l| l.name.as_ref())
+                                                .collect::<Vec<_>>()
+                                                .join(", "),
+                                            assignees: issue
+                                                .assignees
+                                                .iter()
+                                                .map(|u| u.login.as_ref())
+                                                .collect::<Vec<_>>()
+                                                .join(", "),
+                                        })
+                                        .collect();
+
+                                    results
+                                        .entry(name)
+                                        .or_insert(Vec::new())
+                                        .extend(issues_decorator);
+                                }
+                                Err(err) => {
+                                    eprintln!("ERROR: {}", err);
+                                    err.chain()
+                                        .skip(1)
+                                        .for_each(|cause| eprintln!("because: {}", cause));
+                                    std::process::exit(1);
+                                }
                             }
-                            Err(err) => {
-                                eprintln!("ERROR: {}", err);
-                                err.chain()
-                                    .skip(1)
-                                    .for_each(|cause| eprintln!("because: {}", cause));
-                                std::process::exit(1);
+                        }
+
+                        github::QueryKind::Count => {
+                            let count = repository.get_issues_count(&gh, &query).await;
+
+                            match count {
+                                Ok(count) => {
+                                    *context.entry(*name).or_insert(0) += count;
+                                }
+                                Err(err) => {
+                                    eprintln!("ERROR: {}", err);
+                                    err.chain()
+                                        .skip(1)
+                                        .for_each(|cause| eprintln!("because: {}", cause));
+                                    std::process::exit(1);
+                                }
                             }
                         }
-                    }
-                };
+                    };
+                }
             }
         }
 
         for (name, issues) in &results {
-            context.insert(*name, issues);
+            context.insert(format!("{}.{}", repo, name), issues);
```

Comment:
Sorry if we caused you a lot of confusion here but we don't want the `{}.{}` trick :). Just leave the code here as it was before.